recipes-core/images/nilrt-image-common.inc

```diff
@@ -11,4 +11,19 @@ IMAGE_PREPROCESS_COMMAND += "rootfs_update_timestamp;"
 # Do not install license subpackages if they are only recommended.
 BAD_RECOMMENDATIONS_pn-${PN} += "*-lic"
 
+ROOTFS_POSTPROCESS_COMMAND += "\
+       ${@oe.utils.conditional('DISTRO', 'nilrt-nxg', '', 'move_kernel;', d)} \
+"
+
+# Older NILRT distro flavors expect the kernel is an actual image file
+# (e.g. bzImage) and not a symbolic link; fix it up here.
+move_kernel() {
```

Comment:
When we're saying "older NILRT distro flavors" here, we're talking about pre-RAUC images; right? All of the OneRT images are going to use the `nilrt` DISTRO value.

Reply:
Hmm, maybe I should remove the comment altogether since RAUC won't be able to use symlinks in the image either (due to UEFI/FAT32 partition limitation). I'll try to re-word it.